frontend/src/containers/PlayerPage/PlayerPage.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Grid } from '@material-ui/core';
+import ReactPlayer from 'react-player';
+import { withRouter } from 'react-router';
+import TopBar from '../../components/TopBar/TopBar';
+import { DeleteIcon, DownloadIcon, InfoIcon } from '../../assets';
+import './styles.css';
+import { getVideoDetails } from '../../api/VideoAPI';
+
+class PlayerPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      url: undefined,
+      video: undefined,
+    };
+  }
+
+  componentDidMount() {
+    const { match } = this.props;
+    const { videoId } = match.params;
+    getVideoDetails(videoId).then((response) => {
+      const userId = window.sessionStorage.getItem('id');
+      const url = `http://localhost:61346/api/Videos/stream?videoId=${videoId}&userId=${userId}`;
+      this.setState({ url, video: response.data });
+    });
```

Comment:
It would be great to cover case, when this endpoint fails and we get error. One way to cover it, would be by showing error snackbar, and on it's auto hide, we could go back to LibraryPage

Reply:
It would be very nice of course. But you see, my dear friend, any request that is not specifically marked as ranged will just download the whole video which is the thing that we are trying to avoid. I would suggest to just leave it as is for MVP, but if there is a big necessity, we might try to figure something out